variables.tf

```diff
@@ -6,6 +6,11 @@
 variable "role_id" {
   type        = string
   description = "(Required) The camel case role id to use for this role. Cannot contain - characters."
+
+  validation {
+    condition     = can(regex("^[a-z].*[a-zA-Z0-9_.-]*$", var.role_id))
```

Comment:
the condition is not matching the required format ;)

Reply:
please add a suggestion :)

Reply:
resolved